### PR TITLE
Letter 'g' from #428

### PR DESCRIPTION
**Pull request by luzpaz**
Continuing to separate #428 in to separate PRs for each letter. Also removed/corrected some entries that were not relevant/precise and augmented others. This PR has been sorted as well.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4346,39 +4351,56 @@ geoemtries->geometries
...
+gernerate->generate
 gess->guess
 get's->gets
 get;s->gets
 geting->getting
 gettetx->gettext
 Ghandi->Gandhi
+ghraphic->graphic
+gioen->given
+gird->grid
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: The word _gird_ is valid, this correction should be removed.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Removed this entry in favor of https://github.com/lucasdemarchi/codespell/pull/479/files#diff-3bddc99f1f6dffc98ea11be5a5a31ae9R4945

**Comment by larsoner** [comment]
Needs rebase

**Comment by larsoner** [comment]
Also @luzpaz you should be pulling from #436 not #428, right?

**Comment by luzpaz** [comment]
rebased and ready for merge

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -5214,16 +5214,20 @@ gaurenteed->guaranteed
 gaurentees->guarantees
 gausian->gaussian
 gemetrical->geometrical
+gemetry->geometry
 gemoetry->geometry
 genarate->generate
 genarated->generated
 genarating->generating
 genaration->generation
+geneic->generic
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: also "genetic"

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -5249,41 +5254,59 @@ geoemtries->geometries
 geoemtry->geometry
 geographich->geographic
 geographicial->geographical
+geomery->geometry
+geometites->geometries
+geometrc->geometry
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: "geometric" should probably be first

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -5302,32 +5325,43 @@ governmnet->government
...
+greate->greater
 greatful->grateful
 greatfully->gratefully
 greather->greater
 greif->grief
+grephic->graphic
+greysacles->greyscales
 gridles->griddles
 grometry->geometry
+grop->drop
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: or "group"

**Pull request merged**